nodes/src/nodes/router_llm/router.py

```
@@ -0,0 +1,337 @@
+
+    def _route_fallback_chain(self, text: str) -> Dict[str, Any]:
+        """Return the first model in the fallback chain.
+
+        In a real deployment the caller would iterate through the chain
+        on failure.  Here we return the primary model with the full
+        chain attached so downstream code can retry.
+        """
+        with self._lock:
+            self._request_count += 1
+
+        chain = [self.primary_model] + self.fallback_models
```

Ruff (0.15.9)

[warning] Consider `[self.primary_model, *self.fallback_models]` instead of concatenation

Replace with `[self.primary_model, *self.fallback_models]`

(RUF005)